lib/danger/comment_generators/github.md.erb

```diff
@@ -8,10 +8,10 @@
      </tr>
   </thead>
   <tbody>
-    <% table[:content].each do |message| -%>
+    <% table[:content].each do |violation| -%>
     <tr>
       <td>:<%= table[:emoji] %>:</td>
-      <td><%= message %></td>
+      <td data-sticky="<%= violation.sticky %>"><%= violation.message %></td>
```

Comment:
I think it's better to always have it, so in most cases it'll be explicit whether it's sticky. But good point on making it work with older comments!